pkg/limits/frontend/ring_test.go

```diff
@@ -257,7 +257,7 @@ func TestRingStreamUsageGatherer_GetPartitionConsumers(t *testing.T) {
 			AssignedPartitions: map[int32]int64{
 				1: time.Now().Add(-time.Second).UnixNano(),
 			},
-		}, {nil}},
+		}, {AssignedPartitions: nil}},
```

Comment:
We want a `nil` pointer here, rather than a pointer to struct with `nil` map. It doesn't matter much since `getAssignedPartitionsResponseErrs` takes preference.